pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -706,11 +767,30 @@ private void setupPubSub(@NonNull Config cfg, NotificationUtil notificationUtil)
                 }
             };
 
-            Subscriber subscriber = null;
             ProjectSubscriptionName resultSubName = ProjectSubscriptionName.of(projectId, subscriptionId);
-            ExecutorProvider resultsSubExecProvider = InstantiatingExecutorProvider.newBuilder().setExecutorThreadCount(1).build();
-            subscriber = Subscriber.newBuilder(resultSubName, receiver).setParallelPullCount(1).setExecutorProvider(resultsSubExecProvider)
-                    .setMaxAckExtensionPeriod(org.threeten.bp.Duration.ofSeconds(120)).build();
+            ExecutorProvider resultsSubExecProvider = InstantiatingExecutorProvider.newBuilder()
+                .setExecutorThreadCount(1)
+                .build();
+            
+            var subscriberBuilder  = Subscriber.newBuilder(resultSubName, receiver)
+                .setParallelPullCount(1)
+                .setExecutorProvider(resultsSubExecProvider)
+                .setMaxAckExtensionPeriod(org.threeten.bp.Duration.ofSeconds(120));
+
+            if (!StringUtils.isBlank(emulatorHost)) {
```

Comment:
`isBlank` works, but `isEmpty` feels like a better option, behavioral-wise (that is, `pubsub.host = "   "` would result in a URI format option, instead of a `null` return here)